README.md

```diff
@@ -111,23 +111,43 @@ Next / interim revision example:
 
 The generated `.vsix` will be under folder `./dist`
 
-## Publishing vscode-builtin-extensions to open-vsx
+## Packaging builtin-extensions as a builtin-extension-pack '.vsix`
```

Comment:
I see that you've only been following the existing phrasing, but let's take this opportunity to improve this README. I suggest we start naming these extensions: "built-in vscode extensions". The short version could be "built-ins" - no need to use the long form all the time.

The title here would then be something like: "Packaging built-in vscode extensions in an extension pack".

It would make sense to update the whole document, if you are up for it,

Reply:
Done, I updated the whole document as saw it fit, thanks!